AncientReligionsRestored/localization/religion/religion_celtic_l_english.yml

```diff
@@ -44,11 +53,13 @@
  holy_site_somerset_effect_name:0 "From [holy_site|E] #weak ($holy_site_somerset_name$)#!"
  holy_site_wiltshire_name:0 "Stonehenge"
  holy_site_wiltshire_effect_name:0 "From [holy_site|E] #weak ($holy_site_wiltshire_name$)#!"
+ holy_site_london_name:0 "London"
+ holy_site_london_effect_name:0 "From [holy_site|E] #weak ($holy_site_london_name$)#!"
 
 # gaelic
  gaelic_pagan:1 "Gaelic"
  gaelic_pagan_adj:2 "Gaelic"
- gaelic_pagan_desc:1 "The traditional religion of the Celtic people of Ireland, an expansive mythology of heroic god-kings fighting demons and establishing ordered kingdoms. Preserved initially via oral history then recorded by celtic monks, Gaelic belief lives on in the form of folk tales and ancient chronicles."
+ gaelic_pagan_desc:1 "The traditional religion of the Celtic people of Ireland, an expansive mythology of heroic god-kings invading Ireland, fighting demons, and establishing kingdoms. Preserved initially via oral history then recorded by celtic monks, Gaelic belief lives on in the form of folk tales and ancient chronicles."
```

Comment:
*Celtic monks (capitalization)

Reply:
actually, that should say Christian monks. bad find and replace.
good catch!